src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -6,6 +6,8 @@
 
 import org.apache.log4j.Logger;
 import org.usfirst.frc.team467.robot.Drive;
+import org.usfirst.frc.team467.robot.Grabber;
+import org.usfirst.frc.team467.robot.OpticalSensor;
```

Comment:
The only changes to this file is to add imports. That's probably an error, please remove the imports.